ar/index.html

```diff
@@ -31,65 +31,65 @@
 </head>
 <body dir="rtl">
   <main>
-    <h1>!لا تسأل لتسأل، فقط إسأل </h1>
+    <h1>لا تسأل لتسأل، فقط اسأل</h1>
     <p>
-  بين الحين والأخر يأتي احدهم في غرف الدردشة قائلا شيئ مشابه لـ  :  </p>
+      بين الحين والآخر يأتي أحدهم في غرف الدردشة قائلاً شيء مشابه لـ  :
+    </p>
     <blockquote>
       <span class="name">فلان:</span>
       <p class="message">
         هل هناك محترفين جافا هنا؟
       </p>
     </blockquote>
     <p>
-        هذه الطريقة سيئة لأسباب عدة. ما يقصده الشخص
+      هذه الطريقة سيئة، لأسباب عدة. ما يقصده الشخص
       <em>في الحقيقة </em>هو 
     </p>
     <blockquote>
       <span class="name">فلان:</span>
       <p class="message">
- هل هناك اي محترف جافا يقبل مساعدتي في حل المشكلة، مهما كانت، حتى ولو لم تكن لها علاقة ب"جافا" او ان كان هناك اي شخص لا عرف شيئ عن الجافا يستطيع حل مشكلتي؟     </p>
+        هل هناك أي محترف جافا يقبل مساعدتي في حل المشكلة، مهما كانت، حتى ولو لم تكن لها علاقة بجافا أو إن كان هناك أي شخص لايعرف شيئاً عن الجافا يستطيع حل مشكلتي؟
+      </p>
     </blockquote>
     <p>
-  هناك عدة اسباب تجعل الذين لهم خبرة لا يجيبوا عن سؤالك. سؤالك يسأل أكثر مما تظن.
+      هنالك عدة أسباب تجعل الذين يمتلكون الخبرة لايجيبوا عن سؤالك. سؤالك يسأل أكثر مما تظن.
     </p>
     <p>
-  انت تسأل الناس لتحملهم المسؤالية. انت تختبر ثقة الناس في قدراتهم. أنت أيضًا تمنع أشخاصًا آخرين من الإجابة بلا ضرورة.
-
+      أنت تسأل الناس لتحملهم المسؤولية. أنت تختبر ثقة الناس في قدراتهم. أنت أيضاً تمنع أشخاصاً آخرين من الإجابة بلا ضرورة.
     </p>
     <p>
-  يمكن ايضا رؤية سؤالك بهذه الطريقة...
+      بدلاً من ذلك، يمكن رؤية سؤالك على أنه...
```

Comment:
why?

Reply:
The line I've removed - Your question can also be seen this way
The one I've added - Alternatively, your question can be seen as